cli/cli/commands/files/download/download.go

```diff
@@ -53,10 +52,16 @@ const (
 var FilesUploadCmd = &engine_consuming_kurtosis_command.EngineConsumingKurtosisCommand{
 	CommandStr:                command_str_consts.FilesDownloadCmdStr,
 	ShortDescription:          "Download a files artifact from an enclave",
-	LongDescription:           "Download a files artifact using an identifier(name, uuid, shortened uuid) from an enclave to the host machine",
+	LongDescription:           "Download the given files artifact from the given enclave to your machine. The files artifact and enclave are specified by identifier (name, UUID, or shortened UUID). Read more about identifiers here: https://docs.kurtosis.com/reference/resource-identifier",
 	KurtosisBackendContextKey: kurtosisBackendCtxKey,
 	EngineClientContextKey:    engineClientCtxKey,
 	Flags: []*flags.FlagConfig{
+		{
+			Key:     enclaveIdentifierFlagKey,
+			Usage:   "The enclave from which the file will be downloaded. This is a required flag.",
+			Type:    flags.FlagType_String,
+			Default: defaultEnclaveIdentifierKeyword,
```

Comment:
We could probably have introduced a "IsRequired" bool here and no default, but can be done later if/when we see more and more required CLI flag.
Wondering is required CLI flag are an anti-pattern here. I've always considered flags as optional as opposed to args which are usually mandatory. We will see

Reply:
@gbouv agree and I thought of the same thing. But lets see how feedback goes.